db/migrate/20210617062738_create_roles.rb

```diff
@@ -0,0 +1,10 @@
+class CreateRoles < ActiveRecord::Migration[6.0]
+  def change
+    create_table :roles do |t|
+        t.integer :user_id
+        t.string :role
+
```

Comment:
why do we have to put `user_id` here? So let's say we have an `Admin` role and this role is for only one `user`?

Reply:
we currently have 3 roles ma'am: buyer, broker, and admin.
each user_id can only assume 1 role, so thru this table, they will only be assigned one